control/control_validator/src/utils.cpp

```diff
@@ -119,7 +119,7 @@ Trajectory alignTrajectoryWithReferenceTrajectory(
 
   // If last point of predicted_trajectory is behind of end of trajectory, erase points which are
   // behind trajectory last point and insert pNew along the predicted_trajectory
-  // predicted_trajectory:   　　　　p1-----//------pN-2-----pN-1-----pN
+  // predicted_trajectory:           p1-----//------pN-2-----pN-1-----pN
   // trajectory:                     t1-----//-----tN-1--tN
   // ↓
   // predicted_trajectory:           p1-----//------pN-2-pNew
```

Comment:
@kyoichi-sugahara 
is this alignment ok?

Reply:
perfect!!!
zenkaku lol